### DISCUSSION TITLE
JIT artbitrary value syntax for CSS Grid

### DISCUSSION THREAD

**Discussion by uncvrd** [tailwindlabs/tailwindcss#4155] [Help] [answered]
Hey! Quick question, I'm trying to use JIT's built in feature for arbitrary values. One thing that I find a little difficult when reading the docs is understanding the arbitrary value syntax for each class. For example, I'd like to remove a `gridTemplateColumns` variant from my `tailwind.config.js` since it's only used once and use the arbitrary value. What is the syntax to define this inline?

I'd like to convert:

```
gridTemplateColumns: {
  "120px": "repeat(auto-fit, minmax(120px, 1fr))",
},
```

to something like this? But this unfortunately doesn't work:

```
grid-cols-[repeat(auto-fit, minmax(120px, 1fr))]
```

Any help is appreciated, thanks!

**Comment by quick007** [comment, marked as answer]
JIT  doesn't use spaces, remove them and see if it works then. Make sure to mark this as the answer if this helps!

**Reply by simonswiss** [reply]
Yep, that should work fine if you remove the spaces. You can read more on the reason why here:

https://github.com/tailwindlabs/tailwindcss-jit/issues/192#issuecomment-810934401

**Reply by uncvrd** [reply]
that's super nice to know, it works! Thanks both :)

**Comment by mrtnpro** [comment]
Since my issue might fit well in here: What about `col-span-[]` using JIT?
I have tried the following values, without any luck unfortunately:
`grid-span-[17]`, `grid-span-[span 17]`, `grid-span-[span 17/span 17]` and `grid-span-[span 17,span 17]`. None of them seem to generate any classes. Maybe someone can point me in the right direction (if possible at all with JIT) /cc @simonswiss
